### DISCUSSION TITLE
Feature Request: dynamic Openai api key

### DISCUSSION THREAD

**Discussion by keanesnow98** [64bit/async-openai#116] [Ideas]
Hello, currently when creating a new client, it automatically reads openai api key from the environment variable, but this is very inconvenient.
I want to let users input their api keys.
I think this feature should be added to improve library download rate.
Thanks!

```
let client = async_openai::Client::new(); // here
```

**Comment by keanesnow98** [comment]
Sorry, I have not explored all features of this library yet.
I solved like this:

```
let client = async_openai::Client::with_config(async_openai::config::OpenAIConfig::default().with_api_key(openai_api_key));
```

**Comment by 64bit** [comment]
Hi @keanesnow98 

No problem, if you think documentation can be improved PR is welcome!
